src/components/divider/Divider.tsx

```diff
@@ -0,0 +1,17 @@
+import {View} from 'react-native';
+import React from 'react';
+import {useTheme} from '@atb/theme';
+
+export const Divider = () => {
+  const {theme} = useTheme();
+
+  return (
+    <View
+      style={{
+        marginVertical: theme.spacings.medium,
```

Comment:
I don't think the shared `Divider` component should specify its own margin. Margins are situation dependent. I think it is better if it takes in a `style` parameter for the parent to specify margin.